services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx

```diff
@@ -117,13 +119,119 @@ export const ExplosivesPermitForm: FC<ExplosivesPermitFormProps &
+  const [isHistoric, setIsHistoric] = useState<boolean>(!initialValues?.explosives_permit_id && props.isPermitTab);
   const isESUP = props.userRoles.includes(USER_ROLES[Permission.EDIT_EXPLOSIVES_PERMITS]);
   // eslint-disable-next-line no-unused-vars
   const hasEditPermission = isESUP;
   // TODO: See MDS-5201- editing currently disabled
   const disabled = isProcessed; // isProcessed && !hasEditPermission;
+
+  const [radioSelection, setRadioSelection] = useState<number>(props.isPermitTab ? 1 : 2);
+  const [parentView, setParentView] = useState<boolean>(true);
+  const [isAmend, setIsAmend] = useState<boolean>(false);
+  const { isFeatureEnabled } = useFeatureFlag();
+
+  const handleRadioChange = (e) => {
+    setRadioSelection(e.target.value);
+    setIsHistoric(e.target.value == 1);
+    setIsAmend(e.target.value==3);
+  };
+
+  const handleOpenAddExplosivesPermitModal = () => {
+    setParentView(false)
+  }
+
+  const descriptionListElement = (
+    <div>
+      <Typography.Paragraph>
+        <ul className="landing-list">
+          <li>
+            <Typography.Text strong>Add an existing permit </Typography.Text>
+            <Typography.Text>
+              that was previously issued but does not exist in CORE and Minespace. This will help you keep track of your
+              past permits and activities.
+            </Typography.Text>
+          </li>
+          <li>
+            <Typography.Text strong>Create a new permit </Typography.Text>
+            <Typography.Text>this is meant for new explosive storage and use permits.</Typography.Text>
+          </li>
+          <li>
+            <Typography.Text strong>Amend an existing permit </Typography.Text>
+            <Typography.Text>
+              that has already been added to CORE and Minespace. This will allow you to make changes to your permit
+              conditions, such as the dates, amount of explosives.</Typography.Text>
+          </li>
+        </ul>
+      </Typography.Paragraph>
+    </div>
+  );
+
+  const amendDescriptionListElement = (
+    <div>
+      To make changes to an existing explosive storage and use permit, follow these steps:
+      <br />
+      <ul className="landing-list">
+        <li>Open the permit that you want to amend from the applications page of the mine in CORE.</li>
+        <li>Click on the “Amend Permit” button at the top right corner of the permit details page.</li>
+        <li>Fill out the amendment form with the required information and documents.</li>
+        <li>Complete the amendment and issue the permit.</li>
+      </ul>
+    </div>
+  );
+
   return (
+    isFeatureEnabled(Feature.ONE_WINDOW_FOR_CREATING_NEW_OR_HISTORICAL_ESUP) && parentView ? (
+    <>
+      <Form layout="vertical">
+        <Typography.Title level={3}>Add Permit</Typography.Title>
+        <div>
+          <Typography.Paragraph>Let's get your permit started, in CORE you can...</Typography.Paragraph>
```

Comment:
I just ran into this with my PR so I gotta check- was this pushed as --no-verify? If so, can it be linted? Generally it doesn't like unescaped ' and " so that was the tip-off.

Reply:
Updated. Please verify.